config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/service-accounts.yml

```diff
@@ -0,0 +1,43 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cc-api-service-account
+  namespace: #@ data.values.system_namespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cc-api-service-account-superuser
+subjects:
+- kind: ServiceAccount
+  name: cc-api-service-account
+  namespace: #@ data.values.system_namespace
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Does the CC-API really need cluster-admin?

Reply:
nope!  here's our follow-up story: https://www.pivotaltracker.com/story/show/171735131

Reply:
Thanks!